options.go

```diff
@@ -309,6 +312,39 @@ func LabelLimitValueSize(size int) Option {
 	}
 }
 
+// SelfEvictDisablePing configures if ringpop should disable actively pinging
+// other members when it evicts itself from the network. It is not advised to
+// disable the ping as it might cause the suspicion time to kick in as chances
+// are high that the node will be shutdown before its self-eviction is gossiped.
+// Only use this setting if you explicitly require no extra network traffic
+// during shutdown.
+func SelfEvictDisablePing(disabled bool) Option {
```

Comment:
No that I think a bit more about it I need to agree with you.

The same functionality can be achieved by putting a negative value in the ping ratio (zero does not work as it will fall back on the default value). To be completely sure I will add a test for a negative ping ratio.